hs_core/views/__init__.py

```diff
@@ -266,6 +268,12 @@ def is_multiple_file_upload_allowed(request, resource_type, *args, **kwargs):
         return HttpResponseRedirect(request.META['HTTP_REFERER'])
 
 
+def get_relevant_tools(request, shortkey, *args, **kwargs):
+    res, _, _ = authorize(request, shortkey, needed_permission=ACTION_TO_AUTHORIZE.VIEW_RESOURCE)
+    relevant_tools = resource_level_tool_urls(res, request)
+    return HttpResponse(json.dumps(relevant_tools))
+
```

Comment:
do you need a try.. catch here? sometimes json.dumps crashes if you give it unfriendly payloads

Reply:
since we control relevant_tools with a straightforward map, I don't think a try/catch is necessary?